MAUI/Picker/populating-items.md

```diff
@@ -126,38 +126,38 @@ Selected item text style can be customized by setting `SfPicker.SelectedTextStyl
 
     this.Picker.SelectedTextStyle.FontSize = 16;
     this.Picker.SelectedTextStyle.FontAttributes = FontAttributes.Bold;
-    this.Picker.SelectedTextStyle.TextColor = Colors.Gray;
+    this.Picker.SelectedTextStyle.TextColor = Colors.White;
 
 {% endhighlight %}
 {% endtabs %}
 
 ### Unselected item customization
 
-Unselected item text style can be customized by setting `SfPicker.TextStyle` property.
+Unselected item [TextStyle](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Picker.PickerBase.html#Syncfusion_Maui_Picker_PickerBase_TextStyle) can be customized by setting `SfPicker.TextStyle` property.
```

Comment:
The unselected item [TextStyle](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Picker.PickerBase.html#Syncfusion_Maui_Picker_PickerBase_TextStyle) can be customized by setting the `SfPicker.TextStyle` property.

Reply:
Content modified.